app/templates/manage/placement/form.hbs

```diff
@@ -28,6 +28,30 @@
       {{/type-ahead}}
     </div>
   </div>
+  <div class="col">
```

Comment:
Can you change the columns to be closer to `col-9` and `col-3` for md and above? The percent field is way too big.

So, I believe the template field col should be `col col-md-9` and the percent col should be`col col-md-3`